src/app/modules/main/wallet_section/accounts/view.nim

```diff
@@ -98,19 +134,35 @@ QtObject:
         watchOnly.add(item)
       else:
         imported.add(item)
-      
+
     self.watchOnly.setItems(watchOnly)
     self.imported.setItems(imported)
     self.generated.setItems(generated)
 
-  proc generateNewAccount*(self: View, password: string, accountName: string, color: string, emoji: string): string {.slot.} =
-    return self.delegate.generateNewAccount(password, accountName, color, emoji)
+    # create a list of imported seeds/default account created from where more accounts can be derived
+    var generatedAccounts: seq[GeneratedWalletItem] = @[]
+    var importedSeedIndex: int = 1
+    for item in items:
+      if item.getWalletType() == "":
+        var generatedAccs: Model = newModel()
+        generatedAccs.setItems(filter(generated, proc (x: Item): bool = cmpIgnoreCase(x.getDerivedFrom(), item.getDerivedFrom()) == 0))
+        generatedAccounts.add(initGeneratedWalletItem("Default", "status", generatedAccs, item.getDerivedFrom()))
+      elif item.getWalletType() == SEED:
+        var generatedAccs1: Model = newModel()
+        generatedAccs1.setItems(filter(generated, proc(x: Item): bool = cmpIgnoreCase(x.getDerivedFrom(), item.getDerivedFrom()) == 0))
```

Comment:
nit, if you import sugar, the syntax could be:

`generated.filter(x => cmpIgnoreCase(x.getDerivedFrom(), item.getDerivedFrom()) == 0)`